packages/metablocks/src/api.js

```diff
@@ -25,8 +39,257 @@ const createUniverse = async ({
   } catch (e) {
     console.log("from api", e);
     throw e;
+  } finally {
+    console.log(universeKey);
+  }
+};
+
+const updateUniverseV1 = async ({
+  connection,
+  wallet,
+  name,
+  description,
+  websiteUrl,
+}) => {
+  const program = getMetaBlocksProgram(connection, wallet);
+
+  const { universeKey, accounts, updateUniverseArgs } =
+    await computeUpdateUniverseParams({
+      usersKey: wallet.publicKey,
+      name: name,
+      description: description,
+      websiteUrl: websiteUrl,
+    });
+
+  try {
+    const tx = await program.rpc.updateUniverseV1(updateUniverseArgs, {
```

Comment:
`updateUniverseArgs` are a map here, but in `createUniverse`, it's a list of positional arguments. 

Do you think we should update `createUniverse` to accept a map? That will make it consistent with rest of the function calls.

Reply:
done